docs/connector-development/connector-builder-ui/maintenance.md

```diff
@@ -0,0 +1,175 @@
+* Nested configurations 
+    * How to identify: Search for patterns like `config.<a config key>.<a nested config key>` or `config[<a config key>][<a nested config key>]`
+    * How to fix: Flatten the configuration and update the spec. An easy way to do this is to merge both keys like `config.<a config key>_<a nested config key>`
+
+If the import failed for something else, please reach out to the extensibility team with the error so that we can prioritize fixes.
+
+### Implementing the Change
+Once the current version of the connector is working in the Connector Builder, you can now implement the change! 
+
+## Merge Into Code
+
+Goal: Make change available in GitHub
+
+Expected output: The updated codebase
+
+Once you can see the behavior change through the Connector Builder, it's time to show this change to everyone. In order to do so, we need to make the updated manifest available to our airbytehq/airbyte repository.
+
+### New Source Setup
+If you are working on a new source, we need to initialize the source in our GitHub repository which requires a bit more work than to just update a source. Doing this will require you to use the terminal and Git CLI. To do this:
+* If you haven't already done it, pull the git repository using `git clone git@github.com:airbytehq/airbyte.git`. This will create a new folder where the code resides. If you already had the repository on your computer, make sure it's up-to-date by running `git checkout master && git pull`. This command might fail if you have in-flight changes. We recommend committing those changes to a specific branch or stashing them before continuing.
+* Go in the root of the repository  
+* Run `cd airbyte-integrations/connector-templates/generator && ./generate.sh` and generate a `Configuration Based Source`
+* Create a document under `docs/integrations/sources/<name>.md` that describes your source
+* Update `airbyte-integrations/connectors/source-<name>/metadata.yml`
+    * Set `registries` to reflect where this connector should be available
+    * Set `releaseDate`
+    * Set `allowedHosts.hosts`. See [Airbyte Connector Acceptance Test suite](../../connector-development/testing-connectors/connector-acceptance-tests-reference.md) for more details
+* [Optional] Add an icon at the path matching the metadata.yaml `icon` field
+
+If you intend to continue the process through the GitHub UI, you will have to make those changes available to GitHub by running: `git checkout -b source-<name>-creation && git add . && git commit -m "Creating the new source" && git push --set-upstream origin source-<name>-creation`
+
+### Make the Change Available
+
+#### Through Git CLI
+In your local repository:
+* Export the manifest from the Connector Builder, remove the `metadata` field and paste everything in `airbyte-integrations/connectors/source-<name>/source_<name>/manifest.yaml`
+* If exists, delete the `airbyte-integrations/connectors/source-<name>/source_<name>/schemas` folder
```

Comment:
These points seem very specific about converting a previously python or custom-component lowcode connector to connector-builder-compatible. I think it makes sense splitting that out because it's not relevant in most cases.